test_tracetools/package.xml

```diff
@@ -40,6 +40,7 @@
   <test_depend>rclcpp</test_depend>
   <test_depend>rclcpp_lifecycle</test_depend>
   <test_depend>rcpputils</test_depend>
+  <test_depend>rmw_implementation_cmake</test_depend>
```

Comment:
I think we need a `find_package(rmw_implementation_cmake REQUIRED)` somewhere in the CMakeLists.txt to get access to `get_available_rmw_implementations`.  I'm guessing it works now because it is inherited via something else, but we may as well be explicit about it.

Reply:
Good catch, I added it.